src/setup-beam.js

```diff
@@ -471,9 +471,11 @@ async function get(url0, pageIdxs) {
 
 function maybePrependWithV(versionToPrepend, specVersion) {
   const digitStart = /^\d+/
-  return digitStart.test(specVersion)
-    ? `v${versionToPrepend}`
-    : versionToPrepend
+  let v = versionToPrepend
```

Comment:
I preferred the way it was, but I couldn't get the linter and formatter to work side-by-side. It didn't seem like a very bad compromise, this change.